integration/addition_gids_test.go

```diff
@@ -31,9 +31,7 @@ import (
 )
 
 func TestAdditionalGids(t *testing.T) {
-	testPodLogDir, err := os.MkdirTemp("/tmp", "additional-gids")
-	require.NoError(t, err)
-	defer os.RemoveAll(testPodLogDir)
```

Comment:
why leave the test directories in tmp?

Reply:
This will be removed by the testing framework.

https://pkg.go.dev/testing#B.TempDir

Reply:
@mikebrow could you take another look when you get the chance?